### PR TITLE
Remove gcputil test file

### LINKED ISSUE #208: gcputil tests are flaky
### TL;DR

Tests are flaky

### Expected behavior

_No response_

### Observed behavior

_No response_

### Environment Details

```markdown
n/a
```

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by shan4993**
resolves #208

### CODE REVIEW

File: timeutil/duration_test.go [hunk 1]
```diff
@@ -50,12 +50,12 @@ func TestHumanDuration(t *testing.T) {
 		{
 			name:  "zero_minutes_seconds",
 			input: 1 * time.Hour,
-			exp:   "1h",
+			exp:   "2h",
```
Comment (dcreey) [inline timeutil/duration_test.go:53]: Please revert.

File: timeutil/duration_test.go [hunk 2]
```diff
@@ -50,12 +50,12 @@ func TestHumanDuration(t *testing.T) {
...
 		},
 	}
 
 	for _, tc := range cases {
-		tc := tc
+					tc := tc
```
Comment (dcreey) [inline timeutil/duration_test.go:58]: Lint failure.

**Review by dcreey — CHANGES_REQUESTED** [review]
Please fix

File: timeutil/duration_test.go (hunk 1 above)
Comment (shan4993) [reply timeutil/duration_test.go:53]: ack

File: timeutil/duration_test.go (hunk 2 above)
Comment (shan4993) [reply timeutil/duration_test.go:58]: ack

**Review by dcreey — APPROVED** [review]

**Pull request merged**